### ISSUE TITLE
Problem installing Brian 2 with Python 2.7

### ISSUE THREAD

**Issue by jarishna** [brian-team/brian2#945]
Using latest (16/5/18) anaconda distribution on a mac with python 2.7.14 and trying to install via conda gives an error:

```
conda install -c brian-team brian2
Solving environment: failed

UnsatisfiableError: The following specifications were found to be in conflict:
  - blaze
  - brian2
Use "conda info <package>" to see the dependencies for each package.
```

any help appreciated!

**Comment by mstimberg** [comment]
Hi. This is similar to #930 I think, your blaze package seems to depend on numpy 1.14 but brian2 requires numpy < 1.14 (for now, this will change with the next release which should happen next week or so). The best workaround at the moment is to install brian2 into its own environment which avoids such problems. Something along the lines of:
```
conda create -c brian-team -n brian2 python=2 brian2 matplotlib jupyter ipython
```
Should give you a `brian2` environment with all the necessary tools which you can then activate with `conda activate brian2` (or `source activate brian2` -- depends on your configuration). See https://conda.io/docs/user-guide/tasks/manage-environments.html for more details on environments.

**Comment by mstimberg** [comment]
This should be fix with the latest release (currently: 2.1.3.1), please re-open this issue if you still have problems.
